### PR TITLE
kam: replace xmlrpc with jsonrpcs

### PR DESCRIPTION
**Pull request by cruzccl**
Port and transport protocol will remain untouched:

- TCP 8000 for kamusers
- TCP 8001 for kamtrunks

### CODE REVIEW

File: library/Ivoz/Kam/Infrastructure/Kamailio/TrunksClient.php [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    }
+
+    /**
+     * @param array $payload
+     * @return int
+     */
+    private function getActiveCalls(array $payload)
+    {
+        $response = $this->sendRequest(
+            'dlg.profile_get_size',
```
Comment (Kaian) [inline library/Ivoz/Kam/Infrastructure/Kamailio/TrunksClient.php]: I would create a constant for this RPC method like all the other actions have

File: microservices/workers/src/Worker/Xmlrpc.php [hunk 2]
```diff
@@ -5,6 +5,7 @@
 use Doctrine\ORM\EntityManagerInterface;
```
Comment (Kaian) [inline microservices/workers/src/Worker/Xmlrpc.php]: This file is called _Xmlrpc.php_ but does not longer use XML at all, doesn't it?

Comment (mmadariaga) [reply microservices/workers/src/Worker/Xmlrpc.php]: You're right. We'll rename them (jobs, etc.) in another PR

File: library/Ivoz/Kam/Infrastructure/Kamailio/TrunksClient.php (hunk 1 above)
Comment (mmadariaga) [reply library/Ivoz/Kam/Infrastructure/Kamailio/TrunksClient.php]: I like it :)

**Pull request merged**